__tests__/tests.sql

```diff
@@ -1,8 +1,12 @@
 \c nc_news_test
 
-    SELECT articles.author,articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url,
-    COUNT (comments.comment_id) AS comment_count
-    FROM articles
-    LEFT JOIN comments ON articles.article_id = comments.article_id
-    GROUP BY articles.article_id
-    ORDER BY articles.created_at DESC;
\ No newline at end of file
+    -- SELECT articles.author,articles.title, articles.article_id, articles.topic, articles.created_at, articles.votes, articles.article_img_url,
```

Comment:
Ideally this test.sql file should be gitignored, because they're not important for the project they are just to help you practise SQL 👀

Reply:
resolved